packages/fhir-health-care-service/src/components/HealthCareDetail/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { CloseOutlined } from '@ant-design/icons';
+import { Button } from 'antd';
+import { HealthcareServiceDetail } from '../../types';
+import lang from '../../lang';
+import { Dictionary } from 'lodash';
+
+export interface HealthCareDetailProps extends HealthcareServiceDetail {
+  onClose?: Function;
+}
+
+const HealthcareDetails = (props: HealthCareDetailProps) => {
+  const { extraDetails, comment, meta, name, active, id, organization, onClose } = props;
+
+  const detail: Dictionary<string | number | JSX.Element[]> = {
+    [lang.HEALTHCARE_NAME]: name,
+    [lang.STATUS]: active ? lang.ACTIVE : lang.INACTIVE,
+    [lang.IDENTIFIER]: id,
+    [lang.LAST_UPDATED_DATE]: meta?.lastUpdated
+      ? new Date(meta.lastUpdated).toLocaleDateString()
+      : '',
+    [lang.ORGANIZATION]: organization ? organization.name : lang.NOORGANIZATION,
+    [lang.COMMENT]: comment ?? lang.NOCOMMENT,
+    [lang.EXTRADETAILS]: extraDetails ?? lang.NO_EXTRADETAILS,
+  };
+  return (
+    <div className="p-4 bg-white">
+      <Button
+        shape="circle"
+        onClick={() => onClose?.()}
```

Comment:
should we enforce these methods?

Reply:
no i dont think so, we should give flexibility to developers to customize the functionality thats why i let it onclose.

More over it a functional part of component part and the component can serve its purpose even without it thus i left it optional